CreateInstBlobTrigger/locations.py

```diff
@@ -0,0 +1,37 @@
+import xml.etree.ElementTree as ET
+import xmltodict
+
+
+class Locations:
+    """Provides lookup of location data based on UKPRN and LOCID"""
+
+    def __init__(self, root):
+        """Build the locations lookup table
+
+        The key is comprised of two parts:
+
+            UKPRN
+            LOCID
+
+        If LOCATION.LOCUKPRN is present, we use that for the UKPRN part
+        otherwise we use LOCATION.UKPRN for the UKPRN part.
+
+        The LOCID part is the LOCATION.LOCID
+
+        We are not sure if this mapping is correct; it may need a
```

Comment:
Surely this comment can now be removed?

Reply:
Yes, confirmed with Catherine this morning this alg okay. 👍